src/pages/docs/messages/annotations.mdx

```diff
@@ -0,0 +1,529 @@
+    console.log(message.summary);
+  }
+});
+```
+</Code>
+
+<Aside data-type="info">
+When using message annotations, normal messages delivered to the [`subscribe()`](/docs/pub-sub#subscribe) listener will have an `action` of `message.create`.
+</Aside>
+
+For more details on summary structure and usage, see [annotation summaries](#annotation-summaries).
+
+### Subscribing to individual annotations <a id="subscribe-individual-annotations" />
+
+For more fine-grained visibility, you can also subscribe directly to individual annotation events using the `annotations.subscribe()` method on a channel. To subscribe to individual annotations, you must request the `ANNOTATION_SUBSCRIBE` [mode](/docs/channels/options#modes).
+
+Annotations delivered to the `annotations.subscribe()` listener will have an `action` of `annotation.create` or `annotation.delete`.
+
+<Code>
+```javascript
+// Create an Ably Realtime client specifying the clientId that will
+// be associated with annotations published by this client
+const realtime = new Ably.Realtime({ key: '{{API_KEY}}', clientId: 'my-client-id' });
+
+// Create a channel in a namespace called `annotations`
+// which has message annotations enabled.
+// Specify the ANNOTATION_SUBSCRIBE mode to enable annotation subscriptions.
+const channel = realtime.channels.get('annotations:example', { modes: ['ANNOTATION_SUBSCRIBE'] });
+
+await channel.annotations.subscribe((annotation) => {  
+  if (annotation.action === 'annotation.create') {
+    console.log(`New ${annotation.type} annotation with name ${annotation.name} from ${annotation.clientId}`);
+  } else if (annotation.action === 'annotation.delete') {
+    console.log(`${annotation.clientId} deleted a ${annotation.type} annotation with name ${annotation.name}`);
+  }
+});
+```
+
+```nodejs
+// Create an Ably Realtime client specifying the clientId that will
+// be associated with annotations published by this client
+const realtime = new Ably.Realtime({ key: '{{API_KEY}}', clientId: 'my-client-id' });
+
+// Create a channel in a namespace called `annotations`
+// which has message annotations enabled.
+// Specify the ANNOTATION_SUBSCRIBE mode to enable annotation subscriptions.
+const channel = realtime.channels.get('annotations:example', { modes: ['ANNOTATION_SUBSCRIBE'] });
+
+await channel.annotations.subscribe((annotation) => {  
+  if (annotation.action === 'annotation.create') {
+    console.log(`New ${annotation.type} annotation with name ${annotation.name} from ${annotation.clientId}`);
+  } else if (annotation.action === 'annotation.delete') {
+    console.log(`${annotation.clientId} deleted a ${annotation.type} annotation with name ${annotation.name}`);
+  }
+});
+```
+</Code>
+
+<Aside data-type='note'>
+Individual annotation events are useful for activity feeds or detailed logging, but for maintaining UI state, summary events are generally more reliable and efficient.
+</Aside>
+
+
+## Annotation types <a id="annotation-types"/>
```

Comment:
Would this work better as part of publishing an annotation considering that's the point you need to set the type?